src/transformers/configuration_utils.py

```diff
@@ -238,6 +239,7 @@ class PretrainedConfig(PushToHubMixin):
     model_type: str = ""
     is_composition: bool = False
     attribute_map: Dict[str, str] = {}
+    _auto_class: Optional[str] = None
```

Comment:
Since there is just one auto class for this would it make sense to set it as default?

Reply:
No, because setting this value means the code of the object will be copied every time you do `save_pretrained`. So it should not be set for the configs of the lib.